### PR TITLE
Edit title to better fit the current marketing

### PR DESCRIPTION
**Pull request by adamscott**
As asked by @MurderVeggie [here](https://chat.godotengine.org/channel/coffee-break?msg=4GzXd7MHihMHZk8ms).

godot-website

### CODE REVIEW

**Review by mhilbrunner — APPROVED** [review]

**Review by Sauermann — APPROVED** [review]

**Comment by Sauermann** [comment]
Should the line breaks be adjusted?
```
Your free,
open-source
game engine
```

**Comment by coppolaemilio** [comment]
I agree with @Sauermann in terms of line break. I would also try to make it into two lines in bigger screens.
```
Your free, open-source
game engine.
```
and 
```
Your free,
open-source
game engine.
```
For mobile

File: pages/home.html [hunk 1]
```diff
@@ -167,9 +167,8 @@
 	<div class="background-image" style="background-image: url()"></div>
 	<div class="wrapper">
 		<div class="copy container">
-			<h1>The game engine you've been waiting&nbsp;for.</h1>
-			<p>The Godot Engine is a free, all-in-one, cross-platform game engine that makes it easy for you to create 2D and
-				3D games.</p>
+			<h1>Your free, open-source game engine.</h1>
```
Comment (Sauermann) [inline pages/home.html]: Could be solved along this idea:

```suggestion
			<h1><span style="white-space: nowrap;">Your free,</span> <span style="white-space: nowrap;">open-source</span> <span style="white-space: nowrap;">game engine.</span></h1>
```

**Comment by adamscott** [comment]
Updated!

image

File: pages/home.html (hunk 1 above)
Comment (adamscott) [reply pages/home.html]: Thanks for the suggestion! I used instead non-breakable hyphen and `<br>`s.

**Comment by adamscott** [comment]
> I agree with @Sauermann in terms of line break. I would also try to make it into two lines in bigger screens.
> 
> ```
> Your free, open-source
> game engine.
> ```
> 
> and
> 
> ```
> Your free,
> open-source
> game engine.
> ```
> 
> For mobile

Wait, I misread. I'll fix it for desktops.

**Comment by adamscott** [comment]
The thing is that I must make the column containing the title and the buttons larger to accommodate "open-source".

image

I'm liking more the text on 3 lines after trying it out, but your call @coppolaemilio!

**Comment by coppolaemilio** [comment]
I like the version with two lines better. The less vertical space we can use, the better. But we can make it later.
Thanks!

**Pull request merged**